<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectWork</class>
 <widget class="QMainWindow" name="ObjectWork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Предметные области</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QTableView" name="tableName"/>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="3" column="0">
       <widget class="QPushButton" name="addSubjectBut">
        <property name="text">
         <string>Добавить предмет</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QComboBox" name="renameSubject"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="addSubject"/>
      </item>
      <item row="1" column="1" rowspan="3">
       <widget class="Line" name="line">
        <property name="midLineWidth">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="renameSubjectBut">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Изменит название</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <widget class="QLabel" name="status">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QComboBox" name="listSubject"/>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="listName">
        <property name="text">
         <string>Список преподавтелей</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="removeSubject">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="listSubjectBut">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Вывести список</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" rowspan="3">
       <widget class="Line" name="line_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="midLineWidth">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="Line" name="line_2">
        <property name="midLineWidth">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Менеджер работы с предметными областями</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="newNameSubject"/>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="credit">
        <property name="text">
         <string>Зачетный?</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" rowspan="2">
       <widget class="QPushButton" name="removeSubjectBut">
        <property name="text">
         <string>Удалить предмет</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="back">
      <property name="text">
       <string>Вернуться</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>355</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
